package.json

```diff
@@ -76,7 +76,8 @@
 		"changelog": "node ./bin/changelog --changelogSrcType='ZENHUB_RELEASE'",
 		"bump-version": "npm run -s install-if-deps-outdated && php ./bin/update-version.php",
 		"wp-env-mysql-port": "node ./docker/wc-admin-wp-env/mysql-port.js",
-		"create-hook-reference": "node ./bin/hook-reference/index.js"
+		"create-hook-reference": "node ./bin/hook-reference/index.js",
+		"tsi": "php ./bin/tsi.php"
```

Comment:
I think we could use `testing-instructions` versus `tsi`, although `tsi` is shorter it doesn't reflect its purpose clearly.

Reply:
Cool :) Yeah `tsi` is a bit confusing. I was just trying to use something that's shorter than `testing-instructions`.